README.md

```diff
@@ -94,53 +66,79 @@ Each graphics API may have unique requirements that must be satisfied before a c
 
 Any interaction between the `XRSession` the graphics API, such as allocating or retrieving textures, will go through this `XRGraphicsBinding` instance, and the exact mechanics of the interaction will typically be API specific. This allows the rest of the WebXR API to be graphics API agnostic and more easily adapt to future advances in rendering techniques.
 
-## Layer source allocation
+## Layer creation
 
-Once an `XRGraphicsBinding` instance has been acquired, it can be used to create backing `XRLayerSource`s for each layer. What form this takes depends on the API being used, but is generally expected to be a texture or other render target expressed using the APIs native interfaces.
+Once an `XRGraphicsBinding` instance has been acquired, it can be used to create a variety of `XRLayer`s. Any layers created by that graphics binding will then be able to query associated GPU resources each frame, generally expected to be a texture or other render target expressed using the APIs native interfaces.
 
-In all cases a `set____LayerSource` method will be called with the layer that the layer source is being created for, as well as any additional information needed to create the appropriate graphics resources. The method will return a promise that will resolve to the associated `XRLayerSource` type once the graphics resources have been created and the layer is ready to begin presenting images provided by the new layer source.
+The various layer types are created with the  `request____Layer` series of methods on the `XRGraphicsBinding` instance. Information about the graphics resources required, such as whether or not to allocate a depth buffer or alpha channel, are passed in at layer creation time and will be immutable for the lifetime of the layer. The method will return a promise that will resolve to the associated `XRLayer` type once the graphics resources have been created and the layer is ready to be displayed.
 
 ```js
 const xrGfx = new XRWebGLGraphicsBinding(xrSession, gl);
+const layer = await xrGfx.requestProjectionLayer({ alpha: false });
+```
+
+Layer types other than an `XRProjectionLayer` must be given an explicit pixel width and height, as well as whether or not the image should be stereo or mono. This is because those properties cannot be inferred from the hardware or layer type as they can with an `XRProjectionLayer`.
```

Comment:
Note, that the projection layer may be backed by different GPU resources. I'd love to see how that gonna work now w/o layer sources.

Reply:
What do you mean by "different"? A different gl context?

Reply:
Textures, framebuffers, texture arrays. But @toji already responded above.